web/src/pages/HomePage/HomePage.tsx

```diff
@@ -65,4 +75,74 @@ const HomePage = () => {
   )
 }
 
+export const IsMilestone = (page: string, milestone: string) => {
```

Comment:
Functions should be camelCase.
```suggestion
export const isMilestone = (page: string, milestone: string) => {
```